### PR TITLE
Env test cases

### PR DESCRIPTION
**Pull request by anuragpandey1115**

### CODE REVIEW

File: src/__tests__/env.spec.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const execa = require("execa")
```
Comment (vivek8690) [inline src/__tests__/env.spec.ts]: If a dependency is not in use, remove it from package.json as well @anuragpandey1115

File: src/__tests__/env.spec.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    })
+  })
+  
+describe("get current enviornment", () => {
+  it("should console current active env", async() => {
+    const program = await bootstrap();
+    await program.parseAsync(['node', './bin/fdk.js', 'env', 'set', '-n', 'fyndx0' ]);
+    await program.parseAsync(['node', './bin/fdk.js', 'env', 'get']);
+    const currentEnv = configStore.get(CONFIG_KEYS.CURRENT_ENV_VALUE);
+        expect(currentEnv).toMatch('fyndx');
```
Comment (vivek8690) [inline src/__tests__/env.spec.ts]: expect is wrong it should be fyndx0 @anuragpandey1115

File: src/__tests__/env.spec.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  })
+})
+
+  describe("get enviornment list", () => {
+    it("should console all env list", async() => {
+        const program = await bootstrap();
+    await program.parseAsync(['node', './bin/fdk.js', 'env', 'set', '-n', 'fyndx0']);
+    await program.parseAsync(['node', './bin/fdk.js', 'env', 'ls']);
+    const currentEnv = configStore.get(CONFIG_KEYS.CURRENT_ENV_VALUE)
+     expect(currentEnv).toMatch('fyndx0');
```
Comment (vivek8690) [inline src/__tests__/env.spec.ts]: check for all supported  env length @anuragpandey1115

**Review by vivek8690 — CHANGES_REQUESTED** [review]

File: src/__tests__/env.spec.ts (hunk 3 above)
Comment (anuragpandey1115) [reply src/__tests__/env.spec.ts]: samjha nhi

**Pull request merged**